cmd/validation_test.go

```diff
@@ -17,3 +19,35 @@ func TestValidateValidRole(t *testing.T) {
 		t.Errorf("%s", err)
 	}
 }
+
+func TestValidateDagDeploymentArgs(t *testing.T) {
+	myTests := []struct {
+		dagDeploymentType, nfsLocation string
+		expectedOutput                 string
+		expectedError                  error
+	}{
+		{dagDeploymentType: "volume", nfsLocation: "test:/test", expectedError: nil},
+		{dagDeploymentType: "image", expectedError: nil},
+	}
+
+	for _, tt := range myTests {
+		actualError := validateDagDeploymentArgs(tt.dagDeploymentType, tt.nfsLocation)
+		assert.NoError(t, actualError, "optional message here")
+	}
+}
+
+func TestValidateDagDeploymentArgsErrors(t *testing.T) {
+	myTests := []struct {
+		dagDeploymentType, nfsLocation string
+		expectedOutput                 string
+		expectedError                  string
+	}{
+		{dagDeploymentType: "volume", expectedError: "please specify nfs location via --nfs-location flag"},
+		{dagDeploymentType: "unknown", expectedError: "please specify correct dag deployment type, one of: image, volume"},
```

Comment:
If you do those other error message updates you'll need to update the tests here too.

Reply:
done.